reflection.md

```diff
@@ -17,3 +19,7 @@ Write a few brief paragraphs that discuss:
   implementing it? (You don't have to write code for this part. You could say
   "I would use extension to..." or "Scala's rules for valid
   identifiers prevent...")
+
+I thought most of the new operators were much easier to use in this design. It feels much more natural to be able to use the operators than to write a bunch of methods. That being said, sometimes sequences of operators are harder to read than names of methods that are very clear what they do.
+
+I think if I were going to try and make it better, I'd want to tackle how you have to put parentheses around every operator to protect order of operations. I think this would be really hard though, because of the way that Scala implements things and they way we've already leveraged Scala to use infix notation - it seems hard to further break it into allowing us to not have to use a lot of parentheses.
```

Comment:
Being able to control the order of operations might be one big advantage of external DSLs, in that case, even though doing so is still very difficult.